Adding query option information to get cred thumbprint info (#14312)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/application-list.md

This method supports the `$count`, `$expand`, `$filter`, `$orderBy`, `$search`, `$select`, and `$top` [OData query parameters](/graph/query-parameters) to help customize the response. Some queries are supported only when you use the **ConsistencyLevel** header set to `eventual` and `$count`. For more information, see [Advanced query capabilities on Azure AD directory objects](/graph/aad-advanced-queries).

> By default the api does not return the keyCredential thumbprint value when listing all applications. The thumbprint value (`key` property) for application keyCredentials object is returned only if keyCredentials are explicitly requested using a $select when listing all applications or when a single application is queried.
> For example, to get public key thumbprint information for all applications, use the following query `$select=id,appId,keyCredentials`.
> 
> The use of $select to get keyCredentials when listing all applications has a throttling limit of 150 requests per minute for every tenant.
> 

Reviewer: ```suggestion

By default, this API doesn't return the value of the **key** thumbprint in the **keyCredentials** property when listing all applications. To retrieve the **key** thumbprint, the **keyCredentials** property must be specified in a `$select` query. For example, `$select=id,appId,keyCredentials`.

The use of `$select` to get **keyCredentials** for applications has a throttling limit of 150 requests per minute for every tenant.

```
